easybuild/easyconfigs/j/jax/jax-0.2.19-foss-2020b.eb

```diff
@@ -34,6 +34,10 @@ local_jax_prebuildopts = "sed -i 's$pathToSed$%s$g' WORKSPACE && " % local_tf_bu
 
 use_pip = True
 
+# running the tests with lots of cores results in test failures because not enough threads can be started,
+# so limit to a reasonable number of cores to use at maximum
+maxparallel = 10
```

Comment:
FTR: This needlessly limits the parallelism for all packages, e.g. jaxlib which now builds slower. We need this option for extensions...

Reply:
I didn't think of that, but setting this only for the `jax `extension may actually work already... I'll check on that.

Reply:
Doesn't work currently, but it's an easy fix, cfr. https://github.com/easybuilders/easybuild-framework/pull/3811

Reply:
fixed in `jax` easyconfig in #13851